android/app/src/main/res/values/styles.xml

```diff
@@ -66,7 +66,7 @@
     </style>
 
     <style name="Theme.SplashScreen" parent="Theme.SplashScreen.Common">
-        <item name="windowSplashScreenAnimatedIcon">@android:drawable/sym_def_app_icon</item>
+        <item name="windowSplashScreenAnimatedIcon">@drawable/bootsplash</item>
```

Comment:
Also curious about this change, is this still necessary?

Reply:
I didn't spot it.